.travis.yml

```diff
@@ -1,8 +1,7 @@
 language: rust
 
 rust:
-  - 1.39.0
-  - stable
+  - nightly-2019-11-06
```

Comment:
this is necessary for rocket?

Reply:
Yeah, otherwise Rocket won't compile. I picked a particular nightly since sometimes they break.